sdk/core/azure-core/src/main/java/com/azure/core/implementation/ImplUtil.java

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.implementation;
+
+import com.azure.core.http.ContentType;
+import com.azure.core.util.logging.ClientLogger;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.nio.charset.StandardCharsets;
+import org.xml.sax.InputSource;
+import javax.xml.transform.OutputKeys;
+import javax.xml.transform.Source;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerException;
+import javax.xml.transform.sax.SAXSource;
+import javax.xml.transform.sax.SAXTransformerFactory;
+import javax.xml.transform.stream.StreamResult;
+
+/**
+ * This class contains utility methods useful for internal use.
+ */
+public class ImplUtil {
```

Comment:
I know this is a utility in implementation but I'm not a fan of calling this `ImplUtil`. This feels like it should go into `LoggingUtil` given that this is used to format logging messages, that is also in implementation so it will be hidden from public API.